src/components/structures/UserSettings.js

```diff
@@ -690,6 +698,18 @@ module.exports = React.createClass({
         </div>;
     },
 
+    _renderClearCache: function() {
+        return <div>
+            <h3>Clear Cache</h3>
+                <div className="mx_UserSettings_section">
+                    <AccessibleButton className="mx_UserSettings_button danger"
```

Comment:
You made a Zelda reference! /me swoons 

It's deleting data, which I think is dangerous, and it will cause a long resync potentially which is bad. I don't have any strong feelings either way.